src/test/java/org/threeten/extra/chrono/TestInternationalFixedChronology.java

```diff
@@ -0,0 +1,794 @@
+        }
+    }
+
+    @Test
+    public void test_era_yearDay_loop () {
+        for (int year = 1; year < 401; year++) {
+            InternationalFixedDate base = InternationalFixedChronology.INSTANCE.dateYearDay (year, 1);
+            assertEquals (year, base.get (ChronoField.YEAR));
+            InternationalFixedEra era = InternationalFixedEra.CE;
+            assertEquals (era, base.getEra ());
+            assertEquals (year, base.get (ChronoField.YEAR_OF_ERA));
+            InternationalFixedDate eraBased = InternationalFixedChronology.INSTANCE.dateYearDay (era, year, 1);
+            assertEquals (eraBased, base);
+        }
+    }
+
+    @Test
+    public void test_prolepticYear_specific () {
+        assertEquals (InternationalFixedChronology.INSTANCE.prolepticYear (InternationalFixedEra.CE, 4), 4);
+        assertEquals (InternationalFixedChronology.INSTANCE.prolepticYear (InternationalFixedEra.CE, 3), 3);
+        assertEquals (InternationalFixedChronology.INSTANCE.prolepticYear (InternationalFixedEra.CE, 2), 2);
+        assertEquals (InternationalFixedChronology.INSTANCE.prolepticYear (InternationalFixedEra.CE, 1), 1);
+    }
+
+    @Test (expectedExceptions = ClassCastException.class)
+    public void test_prolepticYear_badEra () {
+        InternationalFixedChronology.INSTANCE.prolepticYear (IsoEra.CE, 4);
+    }
+
+    @Test
+    public void test_Chronology_eraOf () {
+        assertEquals (InternationalFixedChronology.INSTANCE.eraOf (0), InternationalFixedEra.CE);
+    }
+
+    @Test (expectedExceptions = DateTimeException.class)
+    public void test_Chronology_eraOf_invalid () {
+        InternationalFixedChronology.INSTANCE.eraOf (2);
+    }
+
+    @Test
+    public void test_Chronology_eras () {
+        List<Era> eras = InternationalFixedChronology.INSTANCE.eras ();
+        assertEquals (eras.size (), 1);
+        assertEquals (eras.contains (InternationalFixedEra.CE), true);
+    }
+
+    //-----------------------------------------------------------------------
+    // Chronology.range
+    //-----------------------------------------------------------------------
+    @Test
+    public void test_Chronology_range () {
+        assertEquals (InternationalFixedChronology.INSTANCE.range (ChronoField.DAY_OF_WEEK), ValueRange.of (1, 7));
+        assertEquals (InternationalFixedChronology.INSTANCE.range (ChronoField.DAY_OF_MONTH), ValueRange.of (1, 28, 29));
+        assertEquals (InternationalFixedChronology.INSTANCE.range (ChronoField.DAY_OF_YEAR), ValueRange.of (1, 364, 365));
+        assertEquals (InternationalFixedChronology.INSTANCE.range (ChronoField.MONTH_OF_YEAR), ValueRange.of (1, 13));
+    }
+
+    //-----------------------------------------------------------------------
+    // InternationalFixedDate.range
+    //-----------------------------------------------------------------------
+    @DataProvider (name = "ranges")
+    Object[][] data_ranges () {
+        return new Object[][] {
```

Comment:
You don't have all range tests needed for dealing with the added, non-week/non-month days.